examples/00-fluent/conjugate_heat_transfer.py

```diff
@@ -0,0 +1,874 @@
+# =================================
+
+import csv
+import os
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pyvista as pv
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+import ansys.fluent.visualization as viz  # noqa: F401
+from ansys.fluent.visualization.matplotlib import Plots  # noqa: F401
+from ansys.fluent.visualization.matplotlib import matplot_windows_manager  # noqa: F401
+from ansys.fluent.visualization.pyvista import Graphics, pyvista_windows_manager
+from ansys.fluent.visualization.pyvista.pyvista_windows_manager import PyVistaWindow
+
+###########################################################################
+# Specifying save path
+# ====================
+# * save_path can be specified as Path("E:/", "pyfluent-examples-tests") or
+# * Path("E:/pyfluent-examples-tests") in a Windows machine for example,  or
+# * Path("~/pyfluent-examples-tests") in Linux.
+save_path = Path(pyfluent.EXAMPLES_PATH)
+
+geom_filename = examples.download_file(
+    "cht_fin_htc_new.scdoc",
+    "pyfluent/examples/CHT",
+    save_path=save_path,
+)
+
+#######################
+# Fluent Solution Setup
+# =====================
+
+#########################################
+# Launch Fluent session with meshing mode
+# =======================================
+
+meshing = pyfluent.launch_fluent(
+    product_version="24.2.0",
+    mode="meshing",
+    dimension=3,
+    precision="double",
+    processor_count=4,
+)
+meshing.health_check.check_health()
+
+#############################################################################
+# Start Watertight Geometry Meshing Workflow
+# ==========================================
+
+meshing.workflow.InitializeWorkflow(WorkflowType=r"Watertight Geometry")
+
+meshing.workflow.TaskObject["Import Geometry"].Arguments = dict(FileName=geom_filename)
+
+meshing.workflow.TaskObject["Import Geometry"].Execute()
+
+meshing.workflow.TaskObject["Add Local Sizing"].Execute()
+
+meshing.workflow.TaskObject["Generate the Surface Mesh"].Arguments = dict(
+    {
+        "CFDSurfaceMeshControls": {
+            "MinSize": 0.3,
+            "MaxSize": 1,
+            "ScopeProximityTo": "faces",
+        },
+    }
+)
+meshing.workflow.TaskObject["Generate the Surface Mesh"].Execute()
+
+meshing.workflow.TaskObject["Describe Geometry"].UpdateChildTasks(SetupTypeChanged=True)
+meshing.workflow.TaskObject["Describe Geometry"].Arguments.setState(
+    {
+        r"CappingRequired": r"No",
+        r"InvokeShareTopology": r"No",
+        r"NonConformal": r"Yes",
+        r"SetupType": r"The geometry consists of both fluid and solid regions and/or voids",  # noqa: E501
+    }
+)
+
+meshing.workflow.TaskObject["Describe Geometry"].Execute()
+
+#############################################################################
+# Update Interface Boundaries; Create Region
+# ==========================================
+
+meshing.workflow.TaskObject["Update Boundaries"].Arguments.setState(
```

Comment:
@prmukherj this is one of the interfaces we'd like to improve for enhance meshing

Reply:
yes, agreed.